rpc/client.go

```diff
@@ -62,9 +65,10 @@ func (c *Client) readPump(funcMap map[string]eth.RPCFunc, logger log.TMLogger) {
 	for {
 		_, message, err := c.conn.ReadMessage()
 		if err != nil {
-			logger.Error("websocket client read message error", "err", err)
-			websocket.IsUnexpectedCloseError(err, websocket.CloseGoingAway, websocket.CloseAbnormalClosure)
-			break
+			if websocket.IsUnexpectedCloseError(err, websocket.CloseGoingAway, websocket.CloseAbnormalClosure) {
```

Comment:
The existing code does not make sense, so I changed to what appears to be intended.

Reply:
Ok I moved this fix elsewhere